ledger-core/v2/application/functest/get_info_test.go

```diff
@@ -1,22 +0,0 @@
-// Copyright 2020 Insolar Network Ltd.
-// All rights reserved.
-// This material is licensed under the Insolar License version 1.0,
-// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
-
-// +build functest
-
-package functest
-
-import (
-	"testing"
-
-	"github.com/stretchr/testify/require"
-)
-
-func TestGetInfo(t *testing.T) {
```

Comment:
It works now (jfyi).

Reply:
@ElSacramento, do we need this test? It's not clear from your comment.

Reply:
We have discussed on one of the team meetings, that it's ok to delete not working tests, but this test works. I'm not sure that it's useful now for platform 2.0, but while it works maybe we can leave it? Just a thought.

Reply:
We discussed in Plat 246, that we don't need this contract, so it will be removed in #69

Reply:
took closer look. agree.